druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/LookupService.java

```diff
@@ -211,16 +215,19 @@ public void update(final LookupData lookupData, byte[] value) {
         String valueColumn;
         Optional<DecodeConfig> decodeConfigOptional = Optional.empty();
         ExtractionNamespace extractionNamespace;
+        List<String> overrideHostList;
```

Comment:
why do you want to expose this? extractionNamespace already has getOverrideLookupServiceHostsList() method